src/components/Admin/Doctors/DoctorForm.jsx

```diff
@@ -1,102 +1,247 @@
-          onChange={changeHandler}
-        />
-        <h3>BASIC INFORMATION</h3>
-        <label htmlFor='file'>Image</label>
-        <input
-          type='file'
-          name='image'
-          id='image'
-          value={model.image}
-          placeholder='Enter your username'
-          onChange={changeHandler}
-        />
-        <label htmlFor='file'>Phone</label>
-        <input
-          type='number'
-          name='phone'
-          id='phone'
-          value={model.phone}
-          onChange={changeHandler}
-        />
-        <label htmlFor='facebook'>Social Networks</label>
-        <BiLogoFacebook size={25} className='iconSocial' />
-        <input
-          style={{ paddingLeft: '3rem' }}
-          type='text'
-          name='facebook'
-          id='facebook'
-          value={model.facebook}
-          placeholder='Enter your username'
-          onChange={changeHandler}
-        />
-        <BiLogoLinkedin size={25} className='iconSocial' />
-        <input
-          style={{ paddingLeft: '3rem' }}
-          type='text'
-          name='linkedin'
-          id='linkedin'
-          value={model.linkedin}
-          placeholder='Enter your username'
-          onChange={changeHandler}
-        />
-        <BiLogoTwitter size={25} className='iconSocial' />
-        <input
-          style={{ paddingLeft: '3rem' }}
-          type='text'
-          name='twitter'
-          id='twitter'
-          value={model.twitter}
-          placeholder='Enter your username'
-          onChange={changeHandler}
-        />
-        <BiLogoInstagram size={25} className='iconSocial' />
-        <input
-          style={{ paddingLeft: '3rem' }}
-          type='text'
-          name='instagram'
-          id='instagram'
-          value={model.instagram}
-          placeholder='Enter your username'
-          onChange={changeHandler}
-        />
-
+      <form onSubmit={handleSubmit} >
+        {
+          !loading &&
+          (<>
+            <h3>USER INFORMATION</h3>
+            <label htmlFor='fullName'>Full Name</label>
+            <input
+              type='text'
+              name='fullName'
+              id='fullName'
+              required
+              value={doctor.fullName}
+              onChange={handleChange}
+            />
+            <label htmlFor='email'>Email</label>
+            <input
+              type='email'
+              name='email'
+              id='email'
+              required
+              value={doctor.email}
+              onChange={handleChange}
+            />
+            <label htmlFor='password'>Password</label>
+            <input
+              type='password'
+              name='password'
+              id='password'
+              required
+              value={doctor.password}
+              onChange={handleChange}
+            />
+            <h3>BASIC INFORMATION</h3>
+            <label htmlFor='file'>Image</label>
+            <input
+              // type='file'
```

Comment:
remove commented line.

Reply:
In this sprint we will upload the image using a text input in which we will manually place the url of the image, in the next sprint we will upload the image using the cloudinary storage, and the search for the image will be using a file input.